images/dashboard/build_and_push.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/python
```

Comment:
Can you reuse py/build_and_push_image.py rather than duplicating a lot of that code? 
You can have a shim script if needed to called build_and_push with appropriate arguments.
(I'm trying to get rid of that code duplication and have a set of reusable python functions).

Reply:
Didn't want to lose time on this before the point mentioned above (one or two docker images) is settled.
If we go with 2, I will clean this.